conf.toml

```diff
@@ -16,6 +16,15 @@ OAuthClientResourceID = ""
 [SecretManager.UNIX]
 SecretFilePath = "../secrets.json"
 
+[EncryptionService]
+Type = "Simple"
+
+[EncryptionService.Simple]
+KeySizeBits = 128
+
+[EncryptionService.GCP_KMS]
```

Comment:
we declared above the type to be  "Simple", is this config declaration needed?

Reply:
No, I just included it as a reference

Reply:
could it be commented out then? as it gives the impression it's required.

Reply:
The same applies to the `InstanceManager.GCP` entry below. So far we have put all values, required or not, in this file to make them easier to discover. I'm ok with commenting out the ones that are not required or have default values, but it should be done in a different PR.

Reply:
I see, didn't realize we were following this pattern, discard the comment then.